packages/components/organisms/f-header/src/components/NavLink.vue

```diff
@@ -1,22 +1,25 @@
 <template>
-    <div>
+    <a
+        v-bind="$attrs"
+        :data-trak="JSON.stringify(dataTrak)"
+        :href="href"
+        :class="[{
+            [$style['c-navLink']]: !isCountryLink,
+            [$style['c-navLink--countryLink']]: isCountryLink,
+            [$style['c-nav-list-link--alt']]: isAltColour,
+            [$style['c-navLink--whiteBackground']]: backgroundTheme === 'white',
+            [$style['c-nav-list-link--transparent']]: backgroundTheme === 'transparent'
+        }]">
         <slot name="icon" />
-        <a
-            v-bind="$attrs"
-            :data-trak="JSON.stringify(dataTrak)"
-            :href="href"
-            :class="[
-                $style['c-navLink'],
-                { [$style['c-navLink--borderBottomBelowMid']]: hasBorderBottom },
-                { [$style['c-navLink--borderTopBelowMid']]: hasBorderTop },
-                { [$style['c-navLink--countryLink']]: isCountryLink },
-                { [$style['c-nav-list-link--alt']]: isAltColour },
-                { [$style['c-nav-list-link--transparent']]: backgroundTheme === 'transparent' }
-            ]">
-
-            {{ text }}
-        </a>
-    </div>
+        <span
+            :class="[{
+                [$style['c-navLink--textWithIcon']]: slotPassed & !isCountryLink,
+                [$style['c-navLink--textWithoutIcon']]: !slotPassed,
+                [$style['c-navLink--borderBottomBelowMid']]: hasBorderBottom,
+                [$style['c-navLink--borderTopBelowMid']]: hasBorderTop
```

Comment:
I'd change up the naming slightly on this now it's moved around.

So I'd call this now `c-navLink-text` and then you can apply the default styles being applied to `c-navLink--textWithIcon` currently to that class.

Then you could call the other classes `c-navLink-text-noIcon`, `c-navLink-text--borderBottomBelowMid` and `c-navLink-text--borderTopBelowMid`.

Hope that makes sense!

Reply:
ah yeah that makes sense, I'll change that now 👍